src/main.h

```diff
@@ -39,6 +40,16 @@ extern "C" {
 
 double tarantool_uptime(void);
 
+/**
+ * Path to Tarantool binary of the process.
+ */
+extern char tarantool_path[PATH_MAX];
```

Comment:
Nit: AFAIK you don't need to specify the array length in the declaration. Let's replace with `extern char tarantool_path[]` to eliminate dependency on `limits.h`.

Reply:
Done.